.github/workflows/license-check-workflow.yml

```
@@ -1,52 +1,51 @@
+            - name: Checkout
+              uses: actions/checkout@v4
+              with:
+                  fetch-depth: 2
+
+            - name: Use Node.js ${{ matrix.node }}
+              uses: actions/setup-node@v4
+              with:
+                  node-version: ${{ matrix.node }}
+
+            - name: Use Java ${{ matrix.java }}
+              uses: actions/setup-java@v3
```

actionlint (1.7.12)

[error] the runner of "actions/setup-java@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)